shoes-core/spec/shoes/button_spec.rb

```diff
@@ -52,4 +52,10 @@
     subject { Shoes::Button.new(app, parent, "text", negative_opts, input_block) }
     it_behaves_like "object with negative dimensions"
   end
+
+  describe "no release" do
+    it "doesn't allow releasing despite allowing clicks" do
+      expect { subject.release {} }.to raise_error(NoMethodError)
```

Comment:
hm wouldn't `expect(subject).not_to respond_to(:release)` (might have to be be_respond_to) clearer than checking for throwing a nomethod error?